css/shelfFlags.css

```diff
@@ -15,4 +15,4 @@
   background-color: #ffdbdf;
   font-weight: bold !important;
   color: red !important;
-}
+}
```

Comment:
I do not see what the difference is here and why it is flagged by git as a change.

Reply:
No difference now - it's probably only shown as a change because I had committed a version with a comment right after (when I was testing for the shelf colour bug) and git incorrectly marked it as a change again, for some reason.